src/find/mod.rs

```diff
@@ -0,0 +1,476 @@
+mod matchers;
+
+use std::cell::RefCell;
+use std::error::Error;
+use std::io::{Write, stderr, stdout};
+use std::rc::Rc;
+use std::time::SystemTime;
+use walkdir::WalkDir;
+use walkdir::WalkDirIterator;
+
+pub struct Config {
```

Comment:
Fair enough. And I realised I can just use `let config = Config::default()` which is less painful than `let config : Config = Default::default()`